### PR TITLE
Use MathJax for LaTeX rendering

### PR DESCRIPTION
**Pull request by lgeiger**
Include a slimed down version of MathJax similar to [`preview-inline`](https://github.com/mangecoeur/preview-inline) as proposed by @mangecoeur in [#99](https://github.com/nteract/nteract/issues/99)

The file size is `3 MB` in total (mainly the font).

I tested it with `hydrogen` and it works great:
latex

### CODE REVIEW

**Comment by lgeiger** [comment]
I rebased it, because the diff is to large to display ;)
The relevant changes are in [this commit](https://github.com/nteract/transformime-jupyter-transformers/pull/17/commits/22433d56eca1ccae25f4bd9931732cdd02065156).

**Comment by minrk** [comment]
Should mathjax be included, and not a dependency?

**Comment by mangecoeur** [comment]
@minrk Including is better because it's easier to include only what's needed (I explored both for preview-inline). There aren't really any packages on npm or bower which would fit - any 'general purpose' package would by definition include things which aren't needed - and a full MathJax install is a beast because they support many browsers and configurations. Even the subset that jupyter/ipython use is pretty heavy. You go from a 35MB install for the full MathJax, to about 12MB for a semi-slim one like ipython use, to about 3MB when you only need to support Atom/Electron (and i suspect it could be even less with a better understanding of font requirements).

**Comment by minrk** [comment]
Gotcha. It might be worthwhile to make that a package, anyway, even within the nteract org: `mathjax-electron`, etc.

**Comment by mangecoeur** [comment]
I guess that could work, assuming it doesn't add too much administrative overhead. Could be a package or a git submodule.

**Comment by rgbkrk** [comment]
> It might be worthwhile to make that a package, anyway, even within the nteract org: mathjax-electron, etc.

This sounds like a great idea, what do you think @lgeiger? I'd use it outside of classic transformime as well (while still in an electron context).

> I guess that could work, assuming it doesn't add too much administrative overhead. Could be a package or a git submodule.

We'll find that out fairly quickly!

**Comment by lgeiger** [comment]
This sounds like a great idea!

I'm a bit busy at the moment so it may take a while, but I'll give it a try this evening.

**Comment by rgbkrk** [comment]
It can be exactly what you have in here as well, including mathjax-helper.js

**Comment by lgeiger** [comment]
OK I made a new package: [mathjax-electron](https://github.com/nteract/mathjax-electron)

**Comment by lgeiger** [comment]
I couldn't figure out how to unittest the SVG from `MathJax`. Is there a possibility to do this?

**Comment by mangecoeur** [comment]
I was able to do some testing in [preview-inline](https://atom.io/packages/preview-inline) using Atom's testing framework since that gives you access to a DOM page for testing. Basically just test that an element is rendered (i.e. exists after the call). No idea how this works in bare Electron. I think it's reasonable to assume MathJax Just Works since they have their own test suite.

**Comment by rgbkrk** [comment]
You may want to work with [electron-mocha](https://github.com/jprichardson/electron-mocha), that's what we use on nteract/nteract and a few other packages instead of JSDom. Works quite well as a drop-in replacement for `mocha` and it runs in the renderer process.

**Comment by lgeiger** [comment]
Maybe I'll try `electron-mocha` in the future. But for now I agree with @mangecoeur.
I think it's reasonable to assume MathJax works, so testing for the existence of the MathJax script is enough.

**Comment by rgbkrk** [comment]
Now that the phosphor folks have fully divested from transformime, I think it's reasonable for us to go ahead with this even though it says Electron on the tin (it's more "chrome-like" than Electron I suppose).

File: src/latex.transform.js [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-    } else {
-        el.innerHTML = katex.renderToString(latex);
-    }
-    return el;
+export function LaTeXTransform(mimetype, value, document) {
+    var container = document.createElement('script');
+    container.type = 'math/tex';
+    container.innerHTML = value.replace('<br>', '');
+
+    mathjaxHelper.loadMathJax(document);
```
Comment (rgbkrk) [inline src/latex.transform.js]: Will this reload MathJax every time a LaTeXTransform is done?

**Comment by rgbkrk** [comment]
Merging, let's keep iterating. Thank you for stellar work as always @lgeiger!

File: src/latex.transform.js (hunk 1 above)
Comment (lgeiger) [inline src/latex.transform.js]: It will only load the first time: https://github.com/nteract/mathjax-electron/blob/master/src/mathjax-electron.js#L6

**Comment by lgeiger** [comment]
Oh I'm flattered 😄.

Should we do another minor release?

**Comment by rgbkrk** [comment]
Sounds great to me.

**Pull request merged**